### PR TITLE
Add Femgineer logo

### PR DESCRIPTION
**Pull request by dalrayes**
Adds bronze sponsor section
Adds Femgineer logo and description
Moves sponsorship blurb to bronze section
@jarmstrng

### CODE REVIEW

**Comment by jarmstrng** [comment]
can you add a `.gitignore` file and add `.DS_Store` to it, so that those two files aren't committed?

**Review by jarmstrng — CHANGES_REQUESTED** [review]
it looks like you missed the changes to the trello card :upside_down_face: the logo needs to be in a section titled Childcare, not Bronze, and they don't get text

File: index.html [hunk 1]
```diff
@@ -369,6 +369,15 @@ <h3 class="col-xs-12">Silver Level Sponsors</h3>
...
+	  </div>
+	  <div class="row sponsor-level">
+			<h3 class="col-xs-12">Bronze Level Sponsors</h3>
+	    <div class="col-xs-12 col-sm-6 col-md-4">
+				<div class="logo">
+					<a href="http://femgineer.com/" target="_blank"><img src="assets/img/logos/femgineer-logo.png" /></a>
+					<p><a href="http://femgineer.com" target="_blank">Femgineer</a> is an education company that empowers forward-thinking techies of all backgrounds and promotes inclusivity in the tech industry.</p>
+				</div>
+			</div>
 			<div class="col-md-4">
```
Comment (jarmstrng) [inline index.html]: Can we get the sponsor-us blurb with a `col-md-8` instead of 4, so that it fills the rest of the container?

**Comment by dalrayes** [comment]
@jarmstrng Added gitignore, updated to childcare sponsor and widened the 'sponsor us' box.

**Comment by jarmstrng** [comment]
Hm, it looks like the .DS_Store files are still in the commit, can you remove them?

File: index.html [hunk 2]
```diff
@@ -369,7 +369,16 @@ <h3 class="col-xs-12">Silver Level Sponsors</h3>
...
 			</div>
-			<div class="col-md-4">
+	  </div>
+	  <div class="row sponsor-level">
+			<h3 class="col-xs-12">Childcare Sponsors</h3>
+	    <div class="col-xs-12 col-sm-6 col-md-4">
+				<div class="logo">
+					<a href="http://femgineer.com/" target="_blank"><img src="assets/img/logos/femgineer-logo.png" /></a>
+					<p><a href="http://femgineer.com" target="_blank">Femgineer</a> is an education company that empowers forward-thinking techies of all backgrounds and promotes inclusivity in the tech industry.</p>
+				</div>
```
Comment (jarmstrng) [inline index.html]: Please remove this text, their sponsor level does not include it

**Comment by dalrayes** [comment]
@jarmstrng Deletes DS Store and sponsor description

**Review by jarmstrng — APPROVED** [review]

**Pull request merged**